src/komponenter/alleredeOpprettetTiltak/GodkjennAvtaleMedAlleredeOpprettetTiltak.tsx

```diff
@@ -16,16 +14,23 @@ interface Props {
 const GodkjennMedAlleredeOpprettetTiltak = (props: Props) => {
     const { alleredeRegistrertAvtale, isApen, onLagre, onLukk } = props;
 
-    const cls = BEMHelper('godkjenn-alleredeOpprettet');
+    const cls = BEMHelper('alleredeOpprettetAvtaleModal');
```

Comment:
Åpent spørsmål, men er det på tide å kvitte seg med BEM og kanskje begynne å bruke less-modules? Det er innebygget støtte i vite. Jeg har en test på det i `AvtaleFetcher.module.less`.

Reply:
Jeg kjenner ikke til less-modules og er ikke oppdatert på state of BEM heller. fint tema til tech hjørnet kanskje?

Reply:
Less-moduler lager bare dynamiske klasser, så det har da ingenting å si hva du kaller CSS-klassene for du kommer aldri til å få konflikert. Det gjør det og enklere å scope stil til en modul i tillegg så kan du få strek typing slik at du ser hvilke stil som er brukt hvor:

MinKomponent.module.less:
```
.main {
    color: red;
}
```

MinKomponent.tsx
```
import { main } from './MinKomponent.module.less';

function MinKomponent() {
    return <div className={main}>Hei</div>;
}
```

Reply:
Jeg lager også en modul-less for RadInfo 😁 6b1b76e8b40869ed731802abaf177379166c4047